tests/lazy_lax_test.py

```diff
@@ -26,7 +26,8 @@ def supported_dtypes(dtypes):
 uint_dtypes = supported_dtypes([np.uint32, np.uint64])
 bool_dtypes = [np.bool_]
 default_dtypes = float_dtypes + int_dtypes
-all_dtypes = float_dtypes + complex_dtypes + int_dtypes + bool_dtypes
+number_dtypes = default_dtypes + complex_dtypes
```

Comment:
I'm actually wondering if we should maintain high levels of similarity with lax_tests.py in case we want to merge into JAX core at a later time, so maybe we should stick to copying and pasting tests from there and not make new definitions like this, what do you think?

Reply:
`number_dtypes` is defined like this in lax_numpy_test.py.

Reply:
Done!